### DISCUSSION TITLE
Running ComfyUI on MacOS

### DISCUSSION THREAD

**Discussion by yuval2508** [comfyanonymous/ComfyUI#3942] [General]
Wanted to run ComfyUI on my 4 years old MacBook Pro intel CPU + AMD GPU and was frustrated from the confusing prerequisites to make it work on my platform. So I ran the "windows portable" package with Wine... yes it works out of the box without any settings. Though the prompt executed in 465.56 seconds (for the default example prompt but with a very large XL model), it had no failures at all.
I just executed `wine run_cpu.bat` and after few seconds ComfyUI opened in my default Safari browser. No dockers, no complex scripts and without messing with python environment. All controls and menus works as expected and for the generation process time it appears to be very close to running on the macOS itself.

What do you think? 
I'm interested if anyone tested that and can share his thoughts, and how we can make it run faster.

Thanks

**Comment by jelling** [comment]
> yes it works out of the box without any settings. Though the prompt executed in 465.56 seconds

Was this the first run or for subsequent ones? Asking because the model is loaded on the first run and that typically takes a long time.

**Reply by yuval2508** [reply]
It takes above 300 seconds to run each time, so this is not the result I wished for.
